README.md

```diff
@@ -38,6 +40,7 @@ To set these environment variables, run these commands:
 conda activate pudl-usage-metrics
 conda env config vars set IPINFO_TOKEN="{your_api_key_here}"
 conda env config vars set DAGSTER_HOME="$(pwd)/dagster_home/"
+conda env config vars set DATA_DIR="$(pwd)/data/"
```

Comment:
Huh! I've never set env vars by conda env before... could set DAGSTER_HOME like that too, I guess. Food for thought.

Reply:
See the line above! :)

Reply:
oh duh